scripts/gitlab/lingua.dic

```diff
@@ -0,0 +1,265 @@
+150
+%
+<
+=
+>
+10k
+accessor/MS
+activations
+adversary/SM
+amongst
+annualised
+Apache-2.0/M
+API
+APIs
+aren
+arg/MS
+assignee/SM
+async
+asynchrony
+autogenerated
+backable
+backend/MS
+benchmark/DSMG
+BFT/M
+bitfield/MS
+blake2/MS
+blockchain/MS
+bool
+borked
+BTC/S
+cancelled
+canonicalize/D
+casted
+CentOS
+CLI/MS
+codebase/SM
+codec/SM
+commit/D
+computable
+conclude/UD
+config/MS
+couldn
+crowdfund
+crowdloan/MSG
+crt
+crypto/MS
+customizable/B
+DDoS
+debian/M
+decodable/MS
+deduplicated
+deinitializing
+dequeue/SD
+deregister
+deserialize/G
+didn
+disincentivize/D
+dispatchable/SM
+DMP/SM
+DMQ
+doesn
+DoS
+DOT
+DOTs
+ECDSA
+ed25519
+encodable
+enqueue/D
+entrypoint/MS
+enum/MS
+ERC-20
+ETH/S
+ethereum/MS
+eval/S
+externality/MS
+extrinsic
+extrinsics
+fd
+fedora/M
+fn
+FRAME/MS
+getter/MS
+GiB/S
+GPL/M
+GPLv3/M
+hadn
+Handler/MS
+HMP/SM
+HRMP
+https
+IFF
+impl
+implementer/MS
+implementor/SM
+includable
+include/BG
+inherent
+inherents
+init/S
+initialize/CRG
+initializer
+instantiate/B
+instantiation/SM
+intrinsic
+intrinsics
+invariant/MS
+invariants
+inverter/MS
+io
+IP/S
+isn
+isolate/BG
+iterable
+jaeger/MS
+js
+keccak256/M
+keypair/MS
+keystore/MS
+KSM/S
+kusama/S
+KYC/M
+lib
+libp2p
+lifecycle/MS
+LRU
+malus
+MB/M
+Mbit
+merkle/MS
+metadata/M
+misbehavior/SM
+misbehaviors
+MIT/M
+MMR
+mpsc
+MPSC
+MQC/SM
+msg
+multisig/S
+multivalidator/SM
+mutex
+NFA
+NFT/SM
+NTB
+offboarding
+onboard/DMSG
+oneshot/MS
+onwards
+OOM/S
+others'
+ourself
+overseer/MS
+ownerless
+parablock/MS
+parachain/MS
+parameterize/D
+parathread/MS
+passthrough
+PDK
+peerset/MS
+permission/D
+phragmen
+picosecond/SM
+polkadot/MS
+pov
+pov-block/MS
+PoV/MS
+preconfigured
+preopen
+prepend/G
+prevalidation
+preverify/G
+programmatically
+prometheus/MS
+provisioner/MS
+proxy/G
+PVF/S
+README/MS
+redhat/M
+register/CD
+repo/MS
+reservable
+responder/SM
+retriability
+reverify
+roundtrip/MS
+rpc
+RPC/MS
+runtime/MS
+rustc/MS
+SAFT
+scalability
+sdf
+sending/S
+smth
+spawn/SR
+sr25519
+startup/MS
+stateful
+str
+struct/MS
+subcommand/SM
+subsystem/MS
+subsystems'
+supermajority
+taskmanager/MS
+TCP
+teleport/D
+teleport/RG
+teleportation/SM
+teleporter/SM
+teleporters
+testnet/MS
+th
+timestamp/MS
+TODO/MS
+transitionary
+trie/MS
+trustless/Y
+tuple/SM
+tx
+typesystem
+ubuntu/M
+UDP
+UI
+unfinalize/B
+unfinalized
+union/MSG
+unordered
+unreceived
+unreserve
+unreserving
+unservable/B
+unvested
+uri
+URI
+utilise
+utilize
+v0
+v1
+v2
```

Comment:
Not sure what to do about those, eventually require backticks I guess but ok for now.